src/PanelTraits/Breadcrumb.php

```diff
@@ -0,0 +1,112 @@
+<?php
+
+namespace Backpack\CRUD\PanelTraits;
+
+trait Breadcrumb
+{
+    /*
+    |--------------------------------------------------------------------------
+    |                              BREADCRUMB
+    |--------------------------------------------------------------------------
+    */
+
+    protected $items;
+
+    /**
+     * Parse the word.
+     *
+     * @return string the parsed word.
+     **/
+    protected function parseWord($word)
+    {
+        $words = explode('_', $word);
+        $string = '';
+        foreach ($words as $key => $word) {
+            $string .= $key > 0 ? ' ' . ucfirst($word) : ucfirst($word);
```

Comment:
Should this be configurable via a `callable`? What if I like ALL_UPPER_CASE or snake_case or DiSsA-CaSe?